plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/auth/PinotAuthenticationType.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pinot.auth;
+
+public enum PinotAuthenticationType {
+    NONE,
+    BASIC_INLINE,
```

Comment:
Rename to `PASSWORD` for consistency with other connectors.

Reply:
is it `PASSWORD` not a bit ambiguous? Password authentication could be used OAuth2 as well for example. The refactoring could also allow users to provide credentials in a password file (not implemented) rather than inline the values in config properties or use a

Reply:
We will use `OAUTH2` or `OAUTH2_PASSTHROUGH` for such use case.